app/lib/dns.server.ts

```diff
@@ -113,11 +113,23 @@ export const upsertRecord = async (
 ) => {
   try {
     if (!isNameValid(name, username)) {
-      throw new Error('Invalid name provided');
+      logger.error('Invalid record name provided', {
+        name,
+        username,
+        baseDomian: `${username}.${process.env.ROOT_DOMAIN}`,
```

Comment:
We should leverage the function I wrote in https://github.com/DevelopingSpace/starchart/blob/main/app/models/user.server.ts#L22-L29 to do this, since we need to remove the `.` from a faculty name.  We could move that function to `app/utils.ts` and you could call it from there?

Reply:
Sure, done!